util/src/trie/sectriedb.rs

```diff
@@ -50,7 +50,7 @@ impl<'db> SecTrieDB<'db> {
 }
 
 impl<'db> Trie for SecTrieDB<'db> {
-	fn iter<'a>(&'a self) -> Box<Iterator<Item = (Vec<u8>, &[u8])> + 'a> {
+	fn iter<'a>(&'a self) -> Box<Iterator<Item=TrieItem> + 'a> {
```

Comment:
This is the canonical style used in `std`: https://doc.rust-lang.org/stable/src/core/up/src/libcore/iter.rs.html#2730